cve_bin_tool/sources/__init__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright (C) 2022 Intel Corporation
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+import os
+from abc import ABC, abstractmethod
+
+# database defaults
+DISK_LOCATION_DEFAULT = os.path.join(os.path.expanduser("~"), ".cache", "cve-bin-tool")
```

Comment:
We've switched to pathlib elsewhere when I merged #1714 , so this either should be switched to use pathlib or we should open an issue to do the refactor separately.